### ISSUE TITLE
No options displayed in the UI

### ISSUE THREAD

**Issue by Harvester62** [xrpgame/xyz_plot_script#7]
When selecting this Script, only "Keep -1 for seeds" is displayed as an option, nothing else is available as options or parameters. The Web UI is running with the following arguments and it's updated to commit ([30b1bcc](https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/30b1bcc64e67ad50c5d3af3a6fe1bd1e9553f34e)):

Launching Web UI with arguments: --xformers --disable-safe-unpickle --api --deepdanbooru --allow-code --administrator

I am sure it was working a few days ago because I did translate it in Italian. There are no error messages in the webui-user console. I disabled all the other non-official scripts as well as all the extensions, apart from the Image Browser, in case of conflicts, but without results. I am on Windows 10 Pro 64bit.

**Comment by Zyin055** [comment]
I installed this script for the first time yesterday and have the same visual issue. No options are displayed except "Keep -1 for seeds" with a short grey bar above it.
Screenshot: https://imgur.com/a/F2jvQQ2

**Comment by CoffeeVectors** [comment]
Yeah I was running into this issue as well. I have the latest WebUI and just downloaded this script for the first time today.

**Comment by zer0TF** [comment]
Resolved with a64128c7afb1b841c28ba670e19681dfbd5701b6, please update your script file!

**Comment by Zyin055** [comment]
Downloaded the new version and now get this error in the console when booting up the Web UI:

```
venv "C:\Stuff\AI\Stable Diffusion\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: c5334fc56b3d44976425da2e6d0a303ae96836a1
Installing requirements for Web UI
Launching Web UI with arguments:
Error loading script: xyz_grid.py
Traceback (most recent call last):
  File "C:\Stuff\AI\Stable Diffusion\modules\scripts.py", line 168, in load_scripts
    compiled = compile(text, scriptfile.path, 'exec')
  File "C:\Stuff\AI\Stable Diffusion\scripts\xyz_grid.py", line 70
    <title>xyz_plot_script/xyz_grid.py at master · xrpgame/xyz_plot_script</title>
                                                 ^
SyntaxError: invalid character '·' (U+00B7)

LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
making attention of type 'vanilla' with 512 in_channels
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
making attention of type 'vanilla' with 512 in_channels
Loading weights [81761151] from C:\Stuff\AI\Stable Diffusion\models\Stable-diffusion\sd-v1-5.ckpt
Global Step: 840000
Applying cross attention optimization (Doggettx).
Model loaded.
Loaded a total of 0 textual inversion embeddings.
Embeddings:

Thanks for being a Gradio user! If you have questions or feedback, please join our Discord server and chat with us: https://discord.gg/feTf9x3ZSB
Running on local URL:  http://127.0.0.1:7860

To create a public link, set `share=True` in `launch()`.
```
